jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -460,10 +499,10 @@ class EndpointK @JvmOverloads constructor(
             },
             logger
         )
-        sctpHandler.setSctpManager(sctpManager)
+        sctpHandler.setSctpManager(sctpManager!!)
```

Comment:
Annoying we need `!!` here, but I guess Kotlin can't prove nothing resets it to `null`?

Reply:
Yeah, I think I originally considered nesting all this inside of an `.also` block after creating `SctpManager`, but there's quite a bit so that didn't feel right.  The other option is to create the `SctpManager` and hold it in a local (non-nullable) val which could be used for all this code and also assigned to the member.